frontend/javascripts/maintenance_banner.tsx

```diff
@@ -4,123 +4,141 @@ import {
 } from "admin/admin_rest_api";
 import { Alert } from "antd";
 import FormattedDate from "components/formatted_date";
-import { useFetch, useInterval } from "libs/react_helpers";
-import { sleep } from "libs/utils";
+import { useInterval } from "libs/react_helpers";
+import _ from "lodash";
 import constants from "oxalis/constants";
+import { setNavbarHeightAction } from "oxalis/model/actions/ui_actions";
 import { setActiveUserAction } from "oxalis/model/actions/user_actions";
 import { Store } from "oxalis/singletons";
 import { OxalisState } from "oxalis/store";
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { useSelector } from "react-redux";
 import { MaintenanceInfo } from "types/api_flow_types";
 
 const INITIAL_DELAY = 5000;
-const INTERVAL_TO_FETCH_MAINTENANCES_MS = 60000;
+const INTERVAL_TO_FETCH_MAINTENANCES_MS = 600000; // 10min
```

Comment:
I intentionally increased this duration to 10min. I don't think we need to check this minute by minute. Maintenance occur rather infrequently.

Reply:
It's true that the polling won't return anything for 99% of the time. However, in the worst case, the maintenance starts and the user is only notified about it 10 minutes later (they probably closed the heads-up several days earlier and forgot about it). That might be 10 minutes of thinking WK is broken, which is why I'd avoid this. I see two options for that:
- introduce another interval which merely checks whether the next-upcoming maintenance just turned into the `current` state (no request necessary)
- stick to the 1 minute interval. this small request pales in comparison to what WK requests for data rendering.

I'd be fine with both options.

Reply:
Fair point. I only considered a maintenance ending and not one starting.

```suggestion
const INTERVAL_TO_FETCH_MAINTENANCES_MS = 60000; // 1min
```